ci_framework/roles/hive/vars/main.yml

```diff
@@ -0,0 +1,40 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+# While options found within the vars/ path can be overridden using extra
+# vars, items within this path are considered part of the role and not
+# intended to be modified.
+
+# All variables within this role should have a prefix of "cifmw_hive"
+
+# YQ parser
+cifmw_hive_yq_install_path: "/usr/bin"
```

Comment:
Just a note.

/usr/* except /usr/local normally should not be written by the administrator. 
The files here considered maintained by the distributor via the distribution's package manger. 

We should also try to avoid placing non distribution files by CI on nodes (which considered test subject)  where a file presence can fix or cause potential  issue.

You can ignore this comment now.

Looks like we will need to discuss the rules for nextgen.
Some places people saying even more strict rules than we received back in the days ,  other places everything seams to be allowed.

Reply:
if we're consuming the `cifmw_path` we may be able to inject in ~/bin for instance.

Reply:
Noted. I have moved out the client to a new role which uses `cifmw_path`. Thanks.